.changelog/26716.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/aws_eip: Defaults to default regional `domain` when `vpc` not set
```

Comment:
```suggestion
resource/aws_eip: Defaults to default VPC `domain` when `vpc` not set
```

Reply:
If the region supports EC2-Classic, the domain will be `standard`